Sources/JXKit/JXValue.swift

```diff
@@ -1098,6 +1093,51 @@ extension JXValue {
         self.init(context: context, valueRef: JSObjectMake(context.contextRef, _class, info))
     }
 
+    // Await the return of this value as a JavaScript `Promise`.
+    public func awaitPromise(priority: TaskPriority = .medium) async throws -> JXValue {
```

Comment:
Significant change: I got rid of async eval() calls and switched to this awaitPromise method. eval() variants have multiplied with the introduction of modules and closures and files, and I didn't want yet more async variants of everything. Given that the previous async eval() required you to return a promise, this makes sense to me. Open to other names.

let result = await eval(...).awaitPromise(priority: .low)